### ISSUE TITLE
[BUG] RHDA can not open pom.xml when it is not in default disk (Windows)

### ISSUE THREAD

**Issue by fravelgue** [fabric8-analytics/fabric8-analytics-vscode-extension#727]
**Describe the bug**
RHDA can not open pom.xml when it is not located in default disk. For example, `X:\`
It shows this error:
```
ENOENT: no such file or directory, access 'C:\fravelgue\learn-spring\pom.xml'
```

**To Reproduce**
1. Open pom.xml that is in a different disk to `C:\`. In the screenshot: `X:\fravelgue\learn-spring\pom.xml`
2. See error

**Expected behavior**
The effective-pom.xml is generated, and I am able to analyze dependencies as normal.

**Screenshots**
![20240731_fabric8-analytics-vscode-extension](https://github.com/user-attachments/assets/d3b02272-cb7e-4aca-92ef-8d5456863aa9)

**Logs**
n/a

**VSCode(please complete the following information):**
 - OS: Windows: 11 23H2
 - VSCode version: 1.91.1
 - Dependency Analytics Version: 0.9.5

**Additional context**
It works when using context menu (right button): `Red Hat Dependecy Analytics Report`.

**Comment by johnsteele** [comment]
I tried reproducing this issue on my mac, and I get different error, not sure if related.
I'll keep investigating.

<img width="1450" alt="Pasted Graphic 10" src="https://github.com/user-attachments/assets/9e191cd7-3313-4d89-87b2-9a91332bc493">

**Comment by johnsteele** [comment]
Here's an update:

I've tested the VSCode extension from source, also being tested/referenced is the fabric8-analytics/fabric8-analytics-lsp-server from source, and I'm now no longer getting the previous error on the same package.json file.

So, maybe there's something in master of this repo, or something in master of fabric8-analytics-lsp-server that didn't make its way in the release. I'll keep investigating.

**Comment by PhilipCattanach** [comment]
Hi @johnsteele 
My understanding of this issue is that the VSCode extension is producing an error message when trying to deal with a pom.xml file that is stored on a different disk drive to the default drive. 
Now that you have overcome the issue you described above, have you been able to recreate the original issue?

**Comment by johnsteele** [comment]
Hi @PhilipCattanach 
Yes, I did test on pom.xml and I was not able to reproduce this issue.

**Comment by Strum355** [comment]
I've tried reproducing this on Win10 with 0.9.5 and latest development builds, but it works as expected in both versions. Specifically, I created a test mvn project on my D:\ drive (as opposed to the default C:\ drive). Is there something special about your X:\ drive e.g. its a network drive or something else? @fravelgue

**Comment by fravelgue** [comment]
Yes, `X:\` is a "Dev Drive" but I have tested in normal drive (`D:\`), and I got same error. It's OK when project is in `C:\`.
